ga4gh/datamodel/variants.py

```diff
@@ -903,7 +917,7 @@ def convertLocationHgvsP(self, hgvsp):
 
     def addCDSLocation(self, effect, cdnaPos):
         hgvsC = effect.hgvsAnnotation.transcript
-        if hgvsC != '':
+        if hgvsC != '' and hgvsC is not None:
```

Comment:
This idiom of testing for an empty string  and None is repeated multiple places.  It would be good to have a function to do this.